docs/user/dashboard/dashboard-troubleshooting.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[dashboard-troubleshooting]]
+== Dashboard and visualizations troubleshooting
+++++
+<titleabbrev>Troubleshooting</titleabbrev>
+++++
+
+Refer to the troubleshooting information to find solutions to common dashboard and visualization issues.
+
+[float]
+[[defer-loading-panels-below-the-fold]]
+=== Improve dashboard loading time
+
+To improve the dashboard loading time, enable the *Defer loading panels below the fold* *Lab*, which loads the dashboard panels as they become visible on the dashboard.
+
+. In the toolbar, click *Labs*.
++
+To enable *Labs*, contact your administrator, or configure the <<advanced-options,Advanced Settings>>. 
+
+. To specify how you want to enable the lab, use the following actions:
+
+* *{kib}* &mdash; Enables the lab in {kib}.
+
+* *Browser* &mdash; Enables the lab for your browser, and persists after you close the browser.
+
+* *Session* &mdash; Enables the lab for this browser session, then resets after you close the browser.
+
+. Click *Close*.
+
+[float]
+[[migrate-timelion-app-worksheets]]
+=== Migrate Timelion app worksheets
+
+deprecated::[The *Timelion* app is deprecated in 7.0.0, and is replaced by dashboard features. In 7.16.0 and later, the *Timelion* app is removed from {kib}. Only the *Timelion* app is deprecated. {kib} continues to support the *Timelion* editor visualizations in *Dashboard*, *Visualize Library*, and *Canvas*. To prepare for the removal of *Timelion* app, you must migrate *Timelion* app worksheets to a dashboard.
```

Comment:
The deprecation isn't formatted correctly.  Not sure "Only the Timelion app is deprecated" is needed as it is stated in the first sentence.